pydicom/pixel_data_handlers/util.py

```diff
@@ -750,6 +750,64 @@ def get_expected_length(ds, unit='bytes'):
     return length
 
 
+def get_pixel_id(ds):
```

Comment:
I think the name is a bit misleading - something like `get_pixel_related_tag_ids` (or something shorter - I'm not good with names...) would be more clear, at least to me.
Apart from that, it looks good to me.